test/e2e/proxy.go

```diff
@@ -105,7 +108,9 @@ func proxyContext(version string) {
 			}
 		}(service.Name)
 
-		// Make an RC with a single pod.
+		// Make an RC with a single pod. The 'porter' image is
```

Comment:
the porter image is used infrequently enough (only 2 or 3 times in my search) that I thought it was prudent to detail what it does.